packages/ui/src/ui/pages/query-tracker/Vcs/Vcs.tsx

```diff
@@ -3,10 +3,26 @@ import {VcsHeader} from './VcsHeader';
 import {VcsItemsList} from './VcsItemsList';
 import './Vcs.scss';
 import cn from 'bem-cn-lite';
+import {useSelector} from 'react-redux';
+import {selectConfigError} from '../module/vcs/selectors';
+import {Alert} from '@gravity-ui/uikit';
 
 const block = cn('vsc');
 
 export const Vcs: FC = () => {
+    const configError = useSelector(selectConfigError);
+
+    if (configError)
```

Comment:
Does the error always mean `usSettings.vcsSettings` is not provided?

Reply:
This message is displayed if there are no vcs available in the settings at all. If there is at least 1 configuration, then there will be no block.